src/signals/settings/hooks/useConfirmedCancel.test.js

```diff
@@ -3,60 +3,67 @@
 import { renderHook } from '@testing-library/react-hooks'
 import * as reactRouterDom from 'react-router-dom'
 
-import useConfirmedCancel, { confirmationMessage } from './useConfirmedCancel'
+import { ConfirmationProvider } from 'components/Confirmation/ConfirmationProvider'
+import * as useConfirm from 'components/Confirmation/useConfirm'
+
+import useConfirmedCancel, {
+  confirmationMessage,
+  title,
+} from './useConfirmedCancel'
 
 jest.mock('react-router-dom', () => ({
   __esModule: true,
   ...jest.requireActual('react-router-dom'),
 }))
 
 const navigateMock = jest.fn()
+const isConfirmedMock = jest.fn()
+const origUseConfirm = useConfirm.useConfirm
+
+jest.spyOn(useConfirm, 'useConfirm').mockImplementation(() => {
+  const orig = origUseConfirm()
+  return {
+    ...orig,
```

Comment:
consider requireActual

Reply:
Removed the origUseConfirm, like you discovered it wasn't necessary.